sechub-server/src/main/java/com/daimler/sechub/server/SecHubSystemPropertyInjector.java

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.server;
+
+import javax.annotation.PostConstruct;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import com.daimler.sechub.sharedkernel.MustBeDocumented;
+
+/**
+ * This component injects some special SecHub Spring Boot values into
+ * corresponding system properties. So we can configure those parts in our
```

Comment:
```suggestion
 * corresponding JVM system properties. So we can configure those parts in our
```

Reply:
It should be clear that this are JVM system properties .. but okay...

Reply:
done